client/components/bloganuary-header/index.tsx

```diff
@@ -0,0 +1,39 @@
+import { useTranslate } from 'i18n-calypso';
+import { useDispatch } from 'react-redux';
+import BloganuaryIcon from 'calypso/components/blogging-prompt-card/bloganuary-icon';
+import isBloganuary from 'calypso/data/blogging-prompt/is-bloganuary';
+import { recordTracksEvent } from 'calypso/state/analytics/actions';
+import './style.scss';
+
+const BloganuaryHeader = () => {
+	const dispatch = useDispatch();
+	const translate = useTranslate();
+
+	const trackBloganuaryMoreInfoClick = () => {
+		dispatch( recordTracksEvent( 'reader_bloganuary_more_info_click' ) );
+	};
+
+	if ( ! isBloganuary() ) {
+		return;
+	}
+
+	return (
+		<div className="bloganuary-header">
+			<div>
+				<BloganuaryIcon />
+				<span className="bloganuary-header__title">{ translate( 'Bloganuary' ) }</span>
+			</div>
+			<a
+				href="https://wordpress.com/bloganuary"
```

Comment:
Do we need to localize this url?  ex `https://wordpress.com/${getLocale()}/bloganuary` or something similar?

Reply:
ahh yes I think so